sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -201,7 +198,7 @@ def identifier_from_raw_id(raw_id: str) -> CommunicationIdentifier:
     """
     if raw_id.startswith('4:'):
         return PhoneNumberIdentifier(
-            value='+{}'.format(raw_id[len('4:'):])
+            value='{}'.format(raw_id[len('4:'):])
```

Comment:
do we really need formatting here? we can directly pass `raw_id[len('4:'):]` as a value

Reply:
Fixed.